source/elements/oneMKL/source/domains/rng/device_routines/device-rng-generate-single.rst

```diff
@@ -0,0 +1,68 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_generate_single:
+
+generate_single
```

Comment:
Do we really want to add `generate_single` to the spec? Maybe postpone it?

Reply:
Removed